### PR TITLE
[IMP] accounting: adding requested link to sales doc

### PR DESCRIPTION
**Pull request by ksc-odoo**
PROJECT TASK (request): https://www.odoo.com/odoo/project.task/3974802?cids=3

-- added an anchor to a Contact Form Configuration section (in different_addresses rst)
-- linked that new anchor to a 'Delivery and invoice addresses' section in an accounting doc (customer_addresses)

as requested in the linked task above 👍

### CODE REVIEW

**Comment by ksc-odoo** [comment]
Hey @samueljlieber -- since this is PURELY just the addition of an anchor, and a subsequent admonition, involving 2 separate (aforementioned docs), do you think you can give this a quick Tech Review/Merge at your earliest convenience? It was a requested addition, via the Project Task linked in the Description. Thanks!

File: content/applications/finance/accounting/customer_invoices/customer_addresses.rst [hunk 1]
```diff
@@ -21,6 +21,10 @@ On quotations and sales orders, there are now fields for :guilabel:`Invoice Addr
 contact record, the corresponding field will use that address by default, but any contact's address
 can be used instead.
 
+.. seealso::
+  For more information, refer to the documentation on :ref:`Contact Form Configuration
+  <send_quotations/contact-form-config>`
+
```
Comment (samueljlieber) [inline content/applications/finance/accounting/customer_invoices/customer_addresses.rst:27]: Suggestion to add an inline reference on the contact record too. I think the seealso is a nice add as well. Also needs a period at the end, and to be indented by another space :)
```suggestion
:ref:`contact record <sales/send_quotations/contact-form-config>`, the corresponding field will use
that address by default, but any contact's address can be used instead.

.. seealso::
   For more information, refer to the documentation on :ref:`Contact Form Configuration
   <sales/send_quotations/contact-form-config>`.
```

File: content/applications/sales/sales/send_quotations/different_addresses.rst [hunk 2]
```diff
@@ -17,6 +17,8 @@ next to :guilabel:`Customer Addresses`, and click :guilabel:`Save`.
    :align: center
    :alt: Activate the Customer Addresses setting.
 
+.. _send_quotations/contact-form-config:
```
Comment (samueljlieber) [inline content/applications/sales/sales/send_quotations/different_addresses.rst:20]: Please add the app or module name in custom anchors :)
```suggestion
.. _sales/send_quotations/contact-form-config:
```

**Review by samueljlieber — APPROVED** [review]
Hi @ksc-odoo, thanks for this quick add to link to additional info on contacts & addresses. Approving with suggestions!
..
@robodoo delegate=ksc-odoo

**Review by dade-odoo — APPROVED** [review]

**Comment by samueljlieber** [comment]
@ksc-odoo this change still relevant, if so please rebase and merge :)

**Pull request closed without merging**